### ISSUE TITLE
Allow xlsm file type to be uploaded

### ISSUE THREAD

**Issue by klecuyer** [gctools-outilsgc/gccollab#324]
https://support.gctools-outilsgc.ca/helpdesk/tickets/3221

User is requesting to add xlsm files (Macro Excel file) to our list of allowable files to upload to GCcollab.

**Comment by markwooff** [comment]
Shouldn't be a problem, can go ahead and add the file type to the accepted file extension list in GCcollab.

**Comment by klecuyer** [comment]
@markwooff Can I add this to GCconnex also?

**Comment by markwooff** [comment]
@klecuyer I don't see why not 👍 can be done the same way

**Comment by klecuyer** [comment]
xlsm added to GCconnex & GCcollab

**Comment by klecuyer** [comment]
I have added xlsm in GCconnex admin but am still unable to upload it. I get an error message and can't proceed.
![image](https://user-images.githubusercontent.com/29754185/39885691-3d2c9918-545b-11e8-8817-909f4b25f74a.png)
![image](https://user-images.githubusercontent.com/29754185/39885711-46ee8cc2-545b-11e8-8e1b-09b05b5cb757.png)
@Phanoix any idea what might cause this? I cleared my browser and cookies. @KevinChagnon
